packaging/google-osconfig-agent.spec

```diff
@@ -38,6 +38,7 @@ Contains the OSConfig agent binary and startup scripts
 GOPATH=%{_gopath} CGO_ENABLED=0 %{_go} build -ldflags="-s -w -X main.version=%{_version}" -mod=readonly -o google_osconfig_agent
 
 %install
+install -d %{buildroot}/etc/osconfig
```

Comment:
see my comment on empty dirs above, but also i think you'll need a corresponding entry in %files for this to have any effect

Reply:
removed